### PR TITLE
Update imaging pipeline

### PR DESCRIPTION
**Pull request by paolomassa**
Imaging pipeline updated:

1. New ELUT correction (developed with Säm): correct just the first and the last energy bin size
2. Fixed bug in stix_gtrans32_test_sep_2021.pro (this procedure still needs to be double checked, but this version is more correct than the previous one)
3. Fixed problem of indices in stix_l1_read

### CODE REVIEW

File: stix/idl/processing/vis/stix_l1_read.pro [hunk 1]
```diff
@@ -38,11 +38,11 @@ bkg_ratio = 1.1
...
 ;CFL and det8
-triggers32(8,*)=data.triggers(t2det(8))/(1.+cfl_ratio)*cfl_ratio
-triggers32(7,*)=data.triggers(t2det(7))/(1.+cfl_ratio)
+triggers32(8,*)=data.triggers(t2det(8)-1)/(1.+cfl_ratio)*cfl_ratio
+triggers32(7,*)=data.triggers(t2det(7)-1)/(1.+cfl_ratio)
 ;BKG and det 16
-triggers32(9,*)=data.triggers(t2det(9))/(1.+bkg_ratio)*bkg_ratio
-triggers32(15,*)=data.triggers(t2det(15))/(1.+bkg_ratio)
+triggers32(9,*)=data.triggers(t2det(9)-1)/(1.+bkg_ratio)*bkg_ratio
+triggers32(15,*)=data.triggers(t2det(15)-1)/(1.+bkg_ratio)
```
Comment (grazwegian) [inline stix/idl/processing/vis/stix_l1_read.pro:45]: I don't really follow what you're doing here. When we have the triggers per group then we can estimate the livetime for that detector directly.

**Review by grazwegian — APPROVED** [review]

File: stix/idl/processing/vis/stix_l1_read.pro (hunk 1 above)
Comment (paolomassa) [reply stix/idl/processing/vis/stix_l1_read.pro:45]: Do you mean to use the function 'stx_livetime_fraction' for computing the livetime?

Comment (grazwegian) [reply stix/idl/processing/vis/stix_l1_read.pro:45]: I just meant in general for the livetime - it is a single value per detector group so once you have the triggers for that group the livetime for both detectors is what you calculate.  I think the ratios are more for when we only have incomplete knowledge of the triggers.

Comment (paolomassa) [reply stix/idl/processing/vis/stix_l1_read.pro:45]: Ok, can we do this correction for the next release? I am planning to change the imaging pipeline using the same routines as for spectroscopy

Comment (grazwegian) [reply stix/idl/processing/vis/stix_l1_read.pro:45]: Yes I think that's a good plan.

**Pull request merged**